engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
no please! let's not mix up "types". I don't think it's a good idea to see case statements checking for types. Especially when all are strings and you'll need to use regex to detect an email address.

:-1: